docs/specs/dynamic_modeling/dynamic_modeling.md

```diff
@@ -0,0 +1,132 @@
+# Vehicle Dynamic Modeling
+
+Vehicle dynamic modeling service employs the supervised machine learning algorithm to generate a learning-based vehicle dynamic model, which can be used as the customized dynamic model in the Apollo simulation platform for control-in-the-loop simulation. The usage of the dynamic modeling asks for three main steps: (1) collect the training/test data for learning-based modeling, via a frontend data collection process monitoring system, (2) upload the collected data into cloud by locally build the standard BOS-linked folder, and (3) submit the service command via the online service webpage and expect the modeling results within email notice.
```

Comment:
Add our new entrance point as well, `bce.apollo.auto` for the fuel service.

Reply:
fixed. thanks